src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -2347,5 +2347,16 @@ public boolean isVideo()
             String extension = getFileExtension();
             return extension != null && VIDEO_EXTENSIONS.contains(extension.toLowerCase());
         }
+
+        /**
+         * Whether or not this attachment is tagged as spoiler,
+         * based on {@link #getFileName()}.
+         *
+         * @return True if this attachment is tagged as spoiler
+         */
+        public boolean isSpoiler()
```

Comment:
I would call it `isSpoiled` or `isMarkedAsSpoiler` to be more precise, preferring the first.

Reply:
That would sound really weird. When I myself hear "spoiled" do I think about sentences like "Such a spoiled brat" as spoiled essentially means ruined, corrupted or bad.
So `isSpoiled` would imo translate to `isBad` and therefore wouldn't make sense. I vote for keeping the method name as is, as it's more than clear enough.

Reply:
Yeah, isSpoiler is better.

Reply:
> Yeah, isSpoiler is better.

It is case sensitive.
Only prefixed with SPOILER_ works.